### PR TITLE
Static icons rework

### PR DESCRIPTION
**Pull request by dmutrokot**

### CODE REVIEW

File: src/components/Citadel/Citadel.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ export default function Citadel({ ownerParcels, className}) {
                 <span className={classes.citadelLoadingLine}></span>
                 <span className={classes.citadelLoadingLine}></span>
                 <div className={classes.citadelLoadingInner}>
-                    <img src={CitadelLoading} alt='Citadel loader' className={classes.citadelLoadingIcon} />
+                    <GotchiverseGif width={'100%'} height={'100%'} />
```
Comment (dudendy) [inline src/components/Citadel/Citadel.js:167]: let's don't use `{}` without needs - `width='100%' height='100%'`

File: src/components/Gotchi/GotchiImage/GotchiSvg.js [hunk 2]
```diff
@@ -51,10 +51,7 @@ export default function GotchiSvg({ id, size, hideWearables, hideBg }) {
     return (
         <div className={classNames(classes.svgWrapper, hideWearables && 'hide-wearables', hideBg && 'hide-bg' )} style={{ width: size }}>
             {loadingSvg ? (
-                <img
-                    className={classes.svgPlaceholder}
-                    src={gotchiLoading} alt='Gotchi Loading...'
-                />
+                <GotchiLoadingGif width={'100%'} />
```
Comment (dudendy) [inline src/components/Gotchi/GotchiImage/GotchiSvg.js:54]: let's don't use `{}` without needs - `width='100%' height='100%'`

File: src/components/Gotchi/GotchiImage/GotchiSvgByStats.js [hunk 3]
```diff
@@ -49,10 +49,7 @@ export default function GotchiSvgByStats({ gotchi, size }) {
     return (
         <div className={classes.svgWrapper} style={{ width: size }}>
             {loadingSvg ? (
-                <img
-                    className={classes.svgPlaceholder}
-                    src={gotchiLoading} alt='Gotchi Loading...'
-                />
+                <GotchiLoadingGif width={'100%'} />
```
Comment (dudendy) [inline src/components/Gotchi/GotchiImage/GotchiSvgByStats.js:52]: let's don't use `{}` without needs - `width='100%' height='100%'`

related to all Icons imports where we use percentages

File: src/pages/Main/components/Portals/Portals.js [hunk 4]
```diff
@@ -67,12 +67,12 @@ export default function Portals() {
...
-                  src={eegg ? sealedPortal : openedPortal }
-                  className={classes.portalsImage}
-                  onClick={onPortalClick}
-                  alt='Portal'
-                />
+                <div className={classes.portalsImage} onClick={onPortalClick}>
+                    { eegg ?
+                        <H1SealedPortalIcon width={150} height={150} /> :
+                        <H1OpenedPortalGif width={150} height={150} />
+                    }
```
Comment (dudendy) [inline src/pages/Main/components/Portals/Portals.js:74]: add please `()` for ternary in template
```
condition ? (
  true
) : (
  false
)
```

**Review by dudendy — CHANGES_REQUESTED** [review]

File: src/components/Citadel/Citadel.js (hunk 1 above)
Comment (dmutrokot) [reply src/components/Citadel/Citadel.js]: fixed, thanks

File: src/components/Gotchi/GotchiImage/GotchiSvgByStats.js (hunk 3 above)
Comment (dmutrokot) [reply src/components/Gotchi/GotchiImage/GotchiSvgByStats.js]: fixed, thanks

File: src/components/Gotchi/GotchiImage/GotchiSvg.js (hunk 2 above)
Comment (dmutrokot) [reply src/components/Gotchi/GotchiImage/GotchiSvg.js]: fixed, thanks

**Pull request merged**